mgmt/tribe/worker/worker.go

```diff
@@ -332,23 +332,27 @@ func (w worker) loadPlugin(plugin core.Plugin) error {
 			}).Info("plugin not found")
 			continue
 		}
+
 		if resp.StatusCode == 200 {
 			if resp.Header.Get("Content-Type") != "application/x-gzip" {
 				logger.WithField("content-type", resp.Header.Get("Content-Type")).Error("Expected application/x-gzip")
 			}
 			dir, err := ioutil.TempDir("", "")
 			if err != nil {
 				logger.Error(err)
+				resp.Body.Close()
```

Comment:
Should this just be a `defer` right after the `c.TribeRequest()` call? It would get rid of the `Body.Close()` below